spec/features/bootstrap_cluster_feature_spec.rb

```diff
@@ -55,7 +55,9 @@
 
       # means it went to the confirmation page
       expect(page).to have_content("Confirm bootstrap")
-      click_button "Bootstrap cluster"
+      fill_in("External Kubernetes API FQDN", with: "some.url")
```

Comment:
in a dev setup i always saw a default value for the apiserver, so i think this is not strictly necessary, but i'd like to keep it for completeness should the default value change to be empty some day